.travis.yml

```diff
@@ -2,7 +2,7 @@ sudo: false
 
 language: node_js
 node_js:
-  - stable
+  - node
```

Comment:
why do you change this?

Reply:
stable was deprecated long time ago in favor of node

Reply:
https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#specifying-nodejs-versions

Reply:
nice!